pkg/volume/csi/csi_plugin.go

```diff
@@ -59,9 +62,54 @@ func ProbeVolumePlugins() []volume.VolumePlugin {
 // volume.VolumePlugin methods
 var _ volume.VolumePlugin = &csiPlugin{}
 
+type csiDriver struct {
+	driverName     string
+	driverEndpoint string
+}
+
+type csiDriversStore struct {
+	driversMap map[string]csiDriver
+	sync.RWMutex
+}
+
+// csiDrivers map keep track of all registered CSI drivers on the node and their
+// corresponding sockets
+var csiDrivers csiDriversStore
+
+var lm labelmanager.LabelManagerInterface
+
+// RegistrationCallback is called by kubelet's plugin watcher upon detection
+// of a new registration socket opened by CSI Driver registrar side car.
+func RegistrationCallback(pluginName string, endpoint string, versions []string, socketPath string) (error, chan bool) {
+
+	glog.Infof("Callback from kubelet with plugin name: %s endpoint: %s versions: %s socket path: %s",
```

Comment:
Wrap all logs statement in the `log(...)`  function to be consistent with other log statements.  This also emit logs prepended with the `csi` label.

Reply:
Will do